frontend/src/routes/Login.tsx

```diff
@@ -101,9 +112,13 @@ const BackButton: React.FC = () => {
         onClick={() => window.history.back()}
         css={{
             cursor: "pointer",
-            display: "flex",
+            display: "inline-flex",
             alignItems: "center",
             gap: 4,
+            padding: 4,
+            borderRadius: 4,
+            ":hover": { outline: "2px solid var(--grey80)" },
```

Comment:
Not sure I would add the hover effect... mhh. I mean it doesn't hurt I guess but the slightly darker text and cursor change should indicate it enough? Mhh